client/src/global/components/atomic/EntityBox/styles.js

```diff
@@ -15,11 +15,14 @@ export const Container = styled.section`
     breakpoints[120],
     breakpoints[120]
   )};
-  padding-block: var(--EntityBox-Container-padding-block, 0)}
+  padding-block: var(--EntityBox-Container-padding-block, 0);
 `;
 
 export const Background = styled.div`
+  --EntityBox-Background-padding-block: 20px;
+  --EntityBox-Background-padding-inline: ${fluidScale("72px", "20px")};
```

Comment:
Prob simpler to add these values as defaults when you use the vars below rather than define them here.